padglobal/padglobal.py

```diff
@@ -237,8 +237,14 @@ async def _add(self, ctx, command, text, confirm=True):
                 await ctx.send('You cannot alias something to itself.')
                 return
             if self.c_commands[text] in self.c_commands:
-                await ctx.send("You cannot alias an alias")
-                return
+                source = self.c_commands[text]
+                if await confirm_message(ctx, 'You cannot alias to an alias.'
```

Comment:
Can you delete this text so it doesn't seem like an error at all? Just start with `{} is already an alias for {}.`

Reply:
discussed in discord, swapped order